website/docs/d/advanced_clusters.html.markdown

```diff
@@ -42,9 +42,65 @@ data "mongodbatlas_clusters" "example" {
 }
 ```
 
+Example using latest schema with independent (asymmetric) shards in the cluster:
+
+```terraform
+resource "mongodbatlas_advanced_cluster" "example" {
+  project_id     = "<YOUR-PROJECT-ID>"
+  name           = "cluster-test"
+  backup_enabled = false
+  cluster_type   = "SHARDED"
+
+  replication_specs {    # Sharded cluster with 2 asymmetric shards (M30 and M40)
+    region_configs {
+      electable_specs {
+        instance_size = "M30"
+        disk_iops     = 3000
+        node_count    = 3
+        disk_size_gb  = 60
+      }
+      analytics_specs {
```

Comment:
same comment as above to remove analytics_specs to simplify the example

Reply:
removed